content/700-optimize/200-getting-started.mdx

```diff
@@ -27,9 +27,20 @@ Prisma Optimize is intended for use in local environments. Learn more in the [FA
 
 ## 2. Add Optimize to your application
 
-### 2.1. Update Your `schema.prisma` file
+### 2.1. Install the Optimize Prisma Client extension
 
-In the `generator` block of your Prisma schema, add the `tracing` preview feature:
+Install Prisma Client, the Optimize extension, and OpenTelemetry API:
+
+```bash
+npm install @prisma/client@latest @prisma/extension-optimize @opentelemetry/api@^1.8.0
```

Comment:
I don't think installing `@opentelemetry/api` should be necessary here since `@prisma/extension-optimize` depends on it already. cc @millsp @igalklebanov to confirm

Otherwise, if we need this, could we please remove `@^1.8.0`? There's no need to specify an older version that would be installed without it.